lib/utils/MyLocalizations.dart

```diff
@@ -14,15 +14,23 @@ class MyLocalizations {
   }
 
   static Future<void> _loadEnglishTranslations() async {
-    var jsonContent = await rootBundle.loadString('assets/language/en_US.json');
+    var jsonContent = await rootBundle.loadString('assets/language/en.json');
     _englishValues = json.decode(jsonContent);
   }
 
   static Future<MyLocalizations> loadTranslations(Locale locale) async {
     var appTranslations = MyLocalizations(locale);
-    var jsonContent = await rootBundle.loadString(
-      'assets/language/${locale.languageCode}_${locale.countryCode}.json'
-    );
+    late var jsonContent;
+    try {
+      jsonContent = await rootBundle.loadString(
+        'assets/language/${locale.languageCode}_${locale.countryCode}.json'
+      );
+    } catch (_) {
+      jsonContent = await rootBundle.loadString(
+        'assets/language/${locale.languageCode}.json'
+      );
+    }
+
     _localisedValues = json.decode(jsonContent);
```

Comment:
In case jsonContent has not been loaded. pass {} to `_localisedValues`

```dart
_localisedValues = json.decode(jsonContent ?? '{}')
```

Reply:
I changed the variable declaration "String? jsonContent;" to be nullable, but when we call "await rootBundle.loadString()", it returns a non-nullable string so then your code proposal would return this warning

"The operand can't be null, so the condition is always 'true'.
Remove the condition.dartunnecessary_null_comparison"